plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -308,6 +358,12 @@ def main():
         state=dict(type='str', default="present", choices=["absent", "present"]),
         lfs_enabled=dict(default=False, type='bool'),
         username=dict(type='str'),
+        allow_merge_on_skipped_pipeline=dict(default=False, type='bool'),
```

Comment:
```suggestion
        allow_merge_on_skipped_pipeline=dict(type='bool', default=False),
```
Please specify `type` before `default`.